ProcessesApi/V1/Controllers/ProcessesApiController.cs

```diff
@@ -50,5 +53,23 @@ public async Task<IActionResult> GetProcessById([FromRoute] ProcessesQuery query
 
             return Ok(process.ToResponse());
         }
+
+        /// <summary>
+        /// Create a new process
+        /// </summary>
+        /// <response code="201">Process has been created successfully</response>
+        /// <response code="400">Bad Request</response> 
+        /// <response code="500">Internal Server Error</response>
+        [ProducesResponseType(typeof(ProcessesResponse), StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPost]
+        [LogCall(LogLevel.Information)]
+        [Route("{process-name}")]
```

Comment:
Are we not using the value on the route, but only the one in the request body?

Reply:
Yes, we are. I thought that was how it was done, but should be fixed now...?